examples/storage/couchdb/images/readiness_probe/join_cluster.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+"""
+    Usage:  join_cluster.py dbname dbname ...
+"""
+from dns import resolver, exception
+import requests
+import logging
+import socket
+import sys
+import os
+
+
+def main(system_dbs):
+    pod_hostnames = discover_pod_fq_hostnames()
+    logging.warning('Using the discovered host names: {0}'.format(pod_hostnames))
+
+    cluster_admin = (os.environ['CLUSTER_ADM_USER'].rstrip(), os.environ['CLUSTER_ADM_PASS'].rstrip())
+    populate_nodes_db(pod_hostnames, cluster_admin)
+    create_system_dbs(cluster_admin, system_dbs)
+    sys.exit(0)
+
+
+def discover_pod_fq_hostnames():
+    hostnames = []
+    fq_pod_hostname = socket.getfqdn(os.environ['POD_IP'])
+    cluster_host = "{}.{}".format(fq_pod_hostname.split('.')[1],
+                                  fq_pod_hostname.split('.')[2])
+    try:
+        dnslookup = resolver.query(cluster_host, "SRV")
+        for raw_hostname in dnslookup:
+            hostnames.append(str(raw_hostname)[8:-1])
```

Comment:
The magic numbers are icky here. Also, isn't this supposed to return an instance of `dns.resolver.Answer`?